docs/install.rst

```diff
@@ -122,11 +122,15 @@ Add a new BIG-IP device (Device Login)
 
 4. Log in to the BIG-IP device. The device hostname, redundancy state, and config sync state are displayed at the top of the page, along with three tabs: Visibility, L2-L3 Stitching, and L4-L7 Configuration.
 
+5. F5 ACI ServiceCenter v2.9 onwards supports LDAP authentication for BIG-IP login.
```

Comment:
F5 ACI ServiceCenter (v2.9+) supports LDAP authentication for BIG-IP login. Admin LDAP users will be able to use all the features of the application.